src/profile_manager/forms.py

```diff
@@ -18,14 +19,32 @@ class Meta:
                   'visible_to_other_students', 'dark_theme', 'silent_mode', 'custom_stylesheet']
 
     def __init__(self, *args, **kwargs):
+        self.request = kwargs.pop('request', None)
+
         super().__init__(*args, **kwargs)
 
         self.fields['grad_year'] = forms.ChoiceField(
             choices=Profile.get_grad_year_choices()
         )
-        
         self.fields['email'].initial = self.instance.user.email
 
+        user = self.instance.user
+
+        # It is possible that the user registered without an email so instead of using
+        # user.emailaddress_set.get(...), let's just use .first() since it would return
+        # None instead of raising a DoesNotExist error
+        email_address = user.emailaddress_set.filter(email=user.email).first()
+
+        if (user.email and email_address is None) or (email_address and email_address.verified is False):
+            resend_email_url = reverse('profiles:profile_resend_email_verification', args=[self.instance.pk])
+            self.fields['email'].help_text = (
+                '<i class="fa fa-info"></i> Not yet verified. '
```

Comment:
use:
`'<i class="fa fa-ban text-danger" aria-hidden="true"></i> Not yet verified. '`